src/DataIngestor/DataIngestor.csproj

```
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
+  <Sdk Name="Aspire.AppHost.Sdk" Version="9.0.0-rc.1.24511.1" />
```

```suggestion
  <Sdk Name="Aspire.AppHost.Sdk" Version="9.0.0-rc.1.24509.13" />
```

Is this weird that it gets the AppHost SDK? It isn't an apphost?

---

Great point, thanks for noticing.

In the beginning, DataIngestor *was* an apphost. It was an experimental way to run Apache Tika in a container for PDF parsing. There's probably a better way to do that without being an apphost, but in any case it became irrelevant because I replaced Tika with a .NET library, PDFPig.

So all of the Aspire reference should have been removed fro DataIngestor. I've pushed a further commit to clean this up (hope that's OK @jmatthiesen / @DamianEdwards).